Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM golang:1.10.0 AS builder
+FROM golang:1.13.0-alpine AS builder
 
 LABEL stage=server-intermediate
 
 WORKDIR /go/src/github.com/infobloxopen/protoc-gen-atlas-query-validate
 COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -o /out/usr/bin/protoc-gen-atlas-query-validate main.go
 
-FROM infoblox/atlas-gentool:latest AS runner
+FROM infoblox/atlas-gentool:v21.3 AS runner
```

Comment:
As far as I remember the idea behind this was to be able to create gentool with current version of protoc-gen-atlas-query-validate for testing. Pointing to a specific version here may not be good for future.

Reply:
suggest to revert it back to `latest`